### PR TITLE
Add missing information linked_data

### PR DESCRIPTION
**Pull request by tomglvng**

### CODE REVIEW

File: content/concepts/products.md [hunk 1]
```diff
@@ -585,6 +587,65 @@ Note that the `locale` and `scope` properties are all set to `null` in this case
...
 :::
 
+### The `linked_data` format
+::: availability versions=Serenity editions=CE,EE
+:::
+
+The sections below describe the format of the `linked_data` property for `pim_catalog_simpleselect` and `pim_catalog_multiselect` types.
+
+#### Simple select attribute
+Whenever the attribute's type is `pim_catalog_simpleselect`, the `linked_data` field should contain an array.
```
Comment (meganelepalud-akeneo) [inline content/concepts/products.md:597]:
```suggestion
Whenever the attribute type is `pim_catalog_simpleselect`, the `linked_data` field must contain an array.
```

File: content/concepts/products.md [hunk 2]
```diff
@@ -585,6 +587,65 @@ Note that the `locale` and `scope` properties are all set to `null` in this case
...
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+#### Multi select attribute
+Whenever the attribute's type is `pim_catalog_multiselect`, the `linked_data` field should contain an array.
```
Comment (meganelepalud-akeneo) [inline content/concepts/products.md:621]:
```suggestion
Whenever the attribute type is `pim_catalog_multiselect`, the `linked_data` field must contain an array.
```

File: content/concepts/products.md [hunk 3]
```diff
@@ -141,7 +142,8 @@ In this formula:
  - `LOCALE_CODE` is the code of a locale when the attribute is localizable, should be equal to `null` otherwise. [Check some examples here.](#the-locale-and-scope-format)
  - `CHANNEL_CODE` is the code of a channel when the attribute is scopable, should be equal to `null` otherwise. [Check some examples here.](#the-locale-and-scope-format)
  - `DATA_INFORMATION` is the value stored for this attribute for this locale (if attribute is localizable) and this channel (if the attribute is scopable). Its type and format depend on the attribute type. [Check some examples here.](#the-data-format)
-
+ - `LINKED_DATA` containing labels if the attribute is an attribute option (only available in Serenity). [Check some examples here.](#the-linked_data-format)
```
Comment (NolwennP) [inline content/concepts/products.md:145]: Attribute can't "be" an attribution option. It can "have". But I would rephrase this way:
```suggestion
 - `LINKED_DATA` containing labels if the attribute is a simple or multi select. [Check some examples here.](#the-linked_data-format) This property if for now, only available in Serenity.
```

File: content/concepts/products.md [hunk 4]
```diff
@@ -585,6 +587,65 @@ Note that the `locale` and `scope` properties are all set to `null` in this case
...
 :::
 
+### The `linked_data` format
+::: availability versions=Serenity editions=CE,EE
+:::
+
+The sections below describe the format of the `linked_data` property for `pim_catalog_simpleselect` and `pim_catalog_multiselect` types.
+
+#### Simple select attribute
+Whenever the attribute type is `pim_catalog_simpleselect`, the `linked_data` field must contain an array.
```
Comment (NolwennP) [inline content/concepts/products.md:597]: Is this really an array? I see an object, but not an array.

File: content/concepts/products.md [hunk 5]
```diff
@@ -585,6 +587,65 @@ Note that the `locale` and `scope` properties are all set to `null` in this case
...
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+#### Multi select attribute
+Whenever the attribute type is `pim_catalog_multiselect`, the `linked_data` field must contain an array.
```
Comment (NolwennP) [inline content/concepts/products.md:621]: Same here.

File: content/swagger/resources/products/definitions/product.yaml [hunk 6]
```diff
@@ -53,7 +53,7 @@ properties:
               description: Product value
             linked_data:
               type: object
-              description: Object containing labels of attribute options (only when query parameter "with_attribute_options" is set to "true")
+              description: Object containing labels of attribute options (only available in Serenity and when query parameter "with_attribute_options" is set to "true")
```
Comment (NolwennP) [inline content/swagger/resources/products/definitions/product.yaml:56]: You should also specify the properties of this object.

File: content/swagger/resources/products/definitions/product.yaml [hunk 7]
```diff
@@ -53,7 +53,15 @@ properties:
               description: Product value
             linked_data:
               type: object
-              description: Object containing labels of attribute options (only when query parameter "with_attribute_options" is set to "true")
+              x-read-only: true
+              description: Object containing labels of attribute options (only available in Serenity and when query parameter "with_attribute_options" is set to "true")
```
Comment (NolwennP) [inline content/swagger/resources/products/definitions/product.yaml:57]:
```suggestion
              description: Object containing labels of attribute options (only available in Serenity and when query parameter "with_attribute_options" is set to "true"). See <a href='/concepts/products.html#the-linked-data-format'>the `linked_data` format</a> section for more details.
```
Link to check, I'm not sure.

File: content/concepts/products.md [hunk 8]
```diff
@@ -141,7 +142,8 @@ In this formula:
  - `LOCALE_CODE` is the code of a locale when the attribute is localizable, should be equal to `null` otherwise. [Check some examples here.](#the-locale-and-scope-format)
  - `CHANNEL_CODE` is the code of a channel when the attribute is scopable, should be equal to `null` otherwise. [Check some examples here.](#the-locale-and-scope-format)
  - `DATA_INFORMATION` is the value stored for this attribute for this locale (if attribute is localizable) and this channel (if the attribute is scopable). Its type and format depend on the attribute type. [Check some examples here.](#the-data-format)
-
+ - `LINKED_DATA` containing the attribute option labels if the attribute is a simple or multi select. [Check some examples here.](#the-linked_data-format) This property if for now, only available in Serenity.
```
Comment (meganelepalud-akeneo) [inline content/concepts/products.md:145]:
```suggestion
 - `LINKED_DATA` containing the attribute option labels if the attribute is a simple or multi select. [Check some examples here.](#the-linked_data-format) This property is for now, only available in Serenity.
```

File: content/swagger/resources/products/definitions/product.yaml (hunk 7 above)
Comment (meganelepalud-akeneo) [reply content/swagger/resources/products/definitions/product.yaml:57]: See <a href='/concepts/products.html#the-linked_data-format'>the `linked_data` format</a> section for more details.
```suggestion
              description: Object containing labels of attribute options (only available in Serenity and when query parameter "with_attribute_options" is set to "true"). See <a href='/concepts/products.html#the-linked_data-format'>the `linked_data` format</a> section for more details.
```

Testé en local ;)

**Review by meganelepalud-akeneo — APPROVED** [review]
